docs/api/carousel.md

```diff
@@ -1,50 +1,27 @@
 ## Carousel component props
 
+* ```plugins: Func[]|String[]```: An array of plugins which will modify and extend carousel behavior. More details [**here**](https://beghp.github.io/gh-pages-rc-15/docs/plugins/plugins)
```

Comment:
```suggestion
* ```plugins: Func[]|String[]```: An array of plugins which will modify and extend carousel behavior. More details [**here**](https://brainhubeu.github.io/react-carousel/docs/plugins/plugins)
```

Reply:
Same as I mentioned before the link that you provided doesn't exist

Reply:
but it must be replaced before merging